coral/src/app/components/Form.test.tsx

```diff
@@ -258,9 +258,9 @@ describe("Form", () => {
         <NativeSelect<Schema>
           name="formFieldsCustomName"
           labelText="NativeSelect"
-          defaultValue={""}
+          defaultValue={"placeholdervalue"}
```

Comment:
The empty string value was not taken into account any more, needed an actual value